arrays.py

```diff
@@ -704,4 +705,10 @@
     "an on-time train",
     "a unit test",
     "covfefe",
+    "a seal of approval",
+    "an idle fancy",
+    "an SSD",
+    "a retro computer",
+    "a layer of untouched snow",
```

Comment:
I’m not sure this is phrased very well, so improvement suggestions from native speakers are very welcome ;)